tutorials/hello/hello_client/src/app/modules/hello/endpoint.ts

```diff
@@ -0,0 +1,31 @@
+import { BaseEndpoint, ModuleEndpointContext, cryptography } from 'lisk-sdk';
+import { CounterStore, CounterStoreData } from './stores/counter';
+import { MessageStore, MessageStoreData } from './stores/message';
+
+export class HelloEndpoint extends BaseEndpoint {
+	public async getHelloCounter(ctx: ModuleEndpointContext): Promise<CounterStoreData> {
+		const counterSubStore = this.stores.get(CounterStore);
+
+		const helloCounter = await counterSubStore.get(
+			ctx,
+			Buffer.from('hello','utf8'),
+		);
+
+		return helloCounter;
+	}
+
+	public async getHello(ctx: ModuleEndpointContext): Promise<MessageStoreData> {
```

Comment:
I am unable to get the Hello Message.

The `getHello` returns this:
```
{
    "id": "1",
    "jsonrpc": "2.0",
    "result": {
        "error": {
            "message": "Key 2cf24dba24704bb97786e846a1665f6386476e9818ecc5ef1544 does not exist."
        }
    }
}
```
When I request using this:
```
{
    "jsonrpc": "2.0",
    "id": "1",
    "method": "hello_getHello",
    "params": {
        "address":"lsk47jw8xw33dueowscuacehbz2htvffm6pgs4r69"
    }
}
```
Since I don’t know which address was used to create the transaction and where can I find that address, I use the `senderPublicKey` available in the transaction JSON and convert it to an lsk address using this:
`cryptography.address.getLisk32AddressFromPublicKey(Buffer.from(address), 'lsk')`.

I also tried the first account in the `genesis_assets.json` and got the same result.